src/vs/workbench/contrib/notebook/browser/viewModel/codeCellViewModel.ts

```diff
@@ -329,16 +339,16 @@ export class CodeCellViewModel extends BaseCellViewModel implements ICellViewMod
 
 	getHeight(lineHeight: number) {
 		if (this._layoutInfo.layoutState === CellLayoutState.Uninitialized) {
-			const editorHeight = this.estimateEditorHeight(lineHeight);
-			return this.computeTotalHeight(editorHeight, 0, 0);
+			const estimate = this.estimateEditorHeight(lineHeight);
+			return this.computeTotalHeight(estimate.editorHeight, 0, 0);
 		} else {
 			return this._layoutInfo.totalHeight;
 		}
 	}
 
-	private estimateEditorHeight(lineHeight: number | undefined = 20): number {
+	private estimateEditorHeight(lineHeight: number | undefined = 20): { editorHeight: number; hasHorizontalScrolling: boolean } {
 		let hasScrolling = false;
-		if (this.layoutInfo.fontInfo) {
+		if (this.layoutInfo.fontInfo && !this.viewContext.notebookOptions.getLayoutConfiguration().wordWrap) {
```

Comment:
We may want to read `BaseCellEditorOptions.wordWrap` which takes language override into account instead of checking `editor.wordwrap`.

Reply:
I don't think I knew about that class. I can't really do that because that class depends on the notebook editor, but all the layout is at the VM level. Seems like that is just configuration that I should be able to get from here

Reply:
I guess it could be accessed through the ViewContext? Seems like just a backdoor though